pycbc/distributions/angular.py

```diff
@@ -388,6 +405,9 @@ def polar_angle(self):
     def azimuthal_angle(self):
         return self._azimuthal_angle
 
+    def cdfinv(self, param, value):
+        raise NotImplementedError("This distribution has not been "
+                                  "implemented for multinest.")
```

Comment:
You have `cdfinv` defined for `SinAngle` and `UniformAngle`, so you should be able to return this. If `param` is the `polar_angle`, then  you just return `self._polardist.cdfinv(param, value)`; if it's the azimuthal angle you return `self._azimuthaldist.cdfinv(param, value)`.